index.js

```diff
@@ -1,16 +1,24 @@
 /* jshint node: true */
 'use strict';
-
 var JSCSFilter = require('broccoli-jscs');
 var jscsrcBuilder = require('./lib/jscsrc-builder');
 
 module.exports = {
   name: 'ember-suave',
 
-  lintTree: function(type, tree) {
+  lintTree: function (type, tree) {
     var jscsOptions = this.app.options.jscsOptions || {};
     jscsOptions.configPath = jscsrcBuilder(this.project);
-
     return new JSCSFilter(tree, jscsOptions);
+  },
+
+  included: function(app, parentAddon) {
+    var target = (parentAddon || app);
+    this._super.included(target);
```

Comment:
So I just took que from ember-cli-qunit: https://github.com/ember-cli/ember-cli-qunit/blob/master/index.js#L38 but if that's not needed, I'd be happy to just call super.  I can try it out and see.